routers/action-router.js

```diff
@@ -0,0 +1,84 @@
+const router = require('express').Router()
+
+const db = require('../data/helpers/actionModel.js')
+
+
+
+//get
+router.get('/', (req, res) => {
+    console.log(`get from action router`);
+    db.get()
+    .then(actions => {
+        console.log(actions)
+        res.status(200).json(actions)
+    })
+    .catch(err => {
+        res.status(500).json({Error: err})
+    })
+});
+
+router.get('/:id', (req, res) => {
+    console.log(`action get with id`)
+    db.get(req.params.id)
```

Comment:
I try not to pass dot notation directly into a function. at least console log it and at best save it as a new variable, console log THAT, then pass it in